definitions/private-law/json/CaseEvent.json

```diff
@@ -89,5 +89,31 @@
     "SecurityClassification": "Public",
     "ShowSummary": "Y",
     "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "05/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "applicantsDetails",
+    "Name": "Applicant details",
+    "Description": "Details of the applicants in the case",
+    "DisplayOrder": 88,
```

Comment:
This should be 8

Reply:
Once all events are done, we will reorder this. it was intentionally added with higher number